Sources/CardPayments/Models/Card.swift

```diff
@@ -22,7 +22,7 @@ public struct Card {
     public var billingAddress: Address?
     
     /// The expiration year and month, in ISO-8601 `YYYY-MM` date format.
-    private var expiry: String {
+    public var expiry: String {
```

Comment:
Does this need to be public?

Reply:
I just found it convenient since I am copying it into an UpdateSetupTokenQuery. The property names are different from Card in the graphQL endpoint, so I just copied the fields into VaultCard defined inside of the UpdateSetupTokenQuery

Reply:
If something has to be public for the SDK but we don't want merchants accessing it, it needs a `nodoc` to hide it from the merchant-facing public docs

Reply:
Addressed all feedback this round here: ae732e8b2187cf2b561bad7b601c611e573bb23a